pkgs/tools/misc/ethminer/default.nix

```diff
@@ -0,0 +1,64 @@
+{ fetchurl, lib, pkgs, stdenv, fetchFromGitHub, opencl-headers, cmake, jsoncpp, boost, makeWrapper,
+  cudatoolkit, mesa, ethash, opencl-info, ocl-icd, openssl, cli11, pkg-config }:
+
+stdenv.mkDerivation rec {
+  name = "${pkgname}-${version}";
+  pkgname = "ethminer";
+  version = "0.18.0-alpha.3";
+
+  src =
+    fetchFromGitHub {
+      owner = "ethereum-mining";
+      repo = "ethminer";
+      rev = "v${version}";
+      sha256 = "0kcgp5kiir8zncmjfr36nlqs363hm8d2m9fc9ld88bjvqwilz7bz";
+      fetchSubmodules = true;
+    };
+
+  # NOTE: dbus is broken
+  cmakeFlags = [
+    "-DHUNTER_ENABLED=OFF"
+    "-DETHASHCUDA=ON"
+    "-DCMAKE_INSTALL_PREFIX=./install"
```

Comment:
I think that if you remove this line, the installPhase will become unnecessary.

Reply:
sill needs the wrapper with path to nvidia looks like, but was able to eliminate the copy to `$out`